pkg/apis/componentconfig/edgecore/v1alpha1/types.go

```diff
@@ -246,6 +246,12 @@ type EdgeHub struct {
 	// Heartbeat indicates heart beat (second)
 	// default 15
 	Heartbeat int32 `json:"heartbeat,omitempty"`
+	// MessageQPS is the QPS to allow while send message to cloudHub.
+	// DefaultQPS: 10
+	MessageQPS int32 `json:"messageQPS,omitempty"`
+	// MessageBurst is the burst to allow while send message to cloudHub.
+	// DefaultBurst: 20
```

Comment:
comment is not equal to default constants: should be modified to 30 60

Reply:
Thanks for you review, I will update it, add rate limiter for edgehub may break some user edge node when user upgrade edgecore, so the default value is important, so I will set the default value big enough and do some test